docs/usage.rst

```diff
@@ -114,6 +114,11 @@ a :class:`CBORTag` object like so::
 
 This will be ignored on decode and the original data content will be returned.
 
+Indefinite containers
+-----------------
+When ``indefinite_containers=True`` is passed to :func:`dump`/:func:`dumps`, the encoder will encode containers (maps and arrays) as indefinite - the length of the container is not explicitly stored in the CBOR, but a break sign is inserted instead.
+
+
```

Comment:
I'm wondering if this section is warranted; we don't have a special section for every single option.

Reply:
Sure, removed. I was just trying to follow the checklist.

Reply:
It's okay – documenting the parameter in the docstrings is enough in this case.